ui/components/ConfirmationModal.js

```diff
@@ -119,9 +119,26 @@ const styles = (theme) => ({
   },
   tabs : {
     marginLeft : 0,
+    boxShadow : 'rgba(0, 179, 159, 0.4) 0px 4px 4px',
+    backgroundColor : theme.palette.type === 'light' ? '#294957' : '#00B39F',
+    '&.MuiTab-root' : {
+      margin : '0px',
+    },
     "& .MuiTab-root.Mui-selected" : {
       backgroundColor : theme.palette.type == "light"? theme.palette.secondary.modalTabs: theme.palette.secondary.mainBackground,
-    }
+    },
+    '& .MuiTabs-indicator' : {
+      height : '2px',
+      borderBottom : '2px solid #00B39F',
+    },
+  },
+  tab : {
+    padding : '0px',
+    margin : '0px',
+    color : '#FBFBFB',
+    '&.Mui-selected' : {
+      color : theme.palette.type === 'light' ? theme.palette.secondary.iconMain : Colors.keppelGreen,
```

Comment:
why `Colors` not `theme.palette.{...}`

Reply:
because I didn't find the theme reference for "Keppel" as previously suggested by Lee 

> @mikenrowland, thanks for this. 👏 
You'll find a theme color to reference under the name of Keppel. 
_Originally posted by @leecalcote in https://github.com/meshery/meshery/pull/8815#discussion_r1334897251_